File: 1-Iniciante/Java/1009.java
package beecrowd 1009;

import java.io.IOException;
import java.util.Scanner;

	public class BeeCrowdJava 1009{
		public static void main (String[] args){
			Scanner in = new Scanner(System.in);
			String nome = in.nextLine();
			Double salario = in.nextDouble();
			Double vendas = in.nextDouble();
			Double TOTAL = ((vendas * 0.15)+salario);
			System.out.printf("TOTAL = R$ %.2f\n",TOTAL);
				
		}	
	
	}
	

File: 1-Iniciante/Dart/1009.dart
import 'dart:io';

void main(){
  String nome;
  double salario, vendas; 

  nome = stdin.readLineSync();
  salario = double.parse(stdin.readLineSync());
  vendas = double.parse(stdin.readLineSync());
  
  var totalSal = ((vendas*0.15)+salario).toStringAsFixed(2);
  print('TOTAL = R\$ $totalSal');
}